packages/Manager/sample-data/observation_data_function.sql

```diff
@@ -0,0 +1,741 @@
+-- This function has to exist in a seperate file because of how spring handles loading SQL files.
+-- with different tooling this might not be needed.
+
+DROP FUNCTION IF EXISTS pg_temp.insert_observation_data(int);
+
+CREATE FUNCTION pg_temp.insert_observation_data(p_observation_measurement_id int)
```

Comment:
Curious why this is in the `pg_temp` namespace?

Reply:
It was to make it a temp function [see](https://stackoverflow.com/questions/4990622/how-to-create-a-temporary-function-in-postgresql) but I've changed it to not need the function now